docs/incidents/2019-02-06-multiple-file-uploads.md

```diff
@@ -0,0 +1,81 @@
+# 2019-02-06
+
+**Incident Leader: Peter**
+
+## Description
+
+Something went wrong and caused this problem.
+
+## Timeline
+
+All times UTC.
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads being done beforehand
```

Comment:
Still not happy with the wording of this - as parallel file uplaods did not exist as a concept/ticket until 10:42

Reply:
I think Cory and I did mention it during the standup.

Reply:
I've taken out the reason and just said `parked` ... followed by people's interpretation of what parked meant.